src/Modules/OrchardCore.Commerce.Payment.Exactly/Views/ExactlySettings.Edit.cshtml

```diff
@@ -0,0 +1,55 @@
+@using Microsoft.AspNetCore.Html
+@using Microsoft.AspNetCore.Mvc.Localization
+@using OrchardCore
+@using OrchardCore.Commerce.Payment.Exactly.Controllers
+@model OrchardCore.Commerce.Payment.Exactly.Models.ExactlySettings
+
+<p class="alert alert-info">
+    @T["Please be sure to use <a href=\"{0}\">this link</a> to create your exactly account.", "https://application.exactly.com/?utm_source=partner&utm_medium=kirill&utm_campaign=LOMBIQ"]
+    @T["That way <a href=\"https://lombiq.com/\">Lombiq</a> (the steward of the Orchard Core Commerce project) will get a commission on the payment fees, which helps cover some of the development costs of OCC."]
+    @T["This is at no cost to you; the fees you pay are the same either way."]
+</p>
+
+<div class="mb-3">
+    <div>
+        <a id="ExactlySettings__testButton"
+            href="@(Orchard.Action<ExactlyController>(controller => controller.VerifyApi()))"
+           class="btn btn-sm btn-primary">
+            @T["Verify currently saved API configuration"]
```

Comment:
Not in this PR/issue, but something similar would be nice for Stripe too.

Reply:
Ok https://github.com/OrchardCMS/OrchardCore.Commerce/issues/426